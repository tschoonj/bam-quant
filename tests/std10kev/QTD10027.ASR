File: C:\xrfdaten\2014KW30\chris\std10kev\std10027.spe
Start: 21.07.2014 16:02:56
Stop: 21.07.2014 16:04:37
BESSY: 298.0013134 299.0730879
IONI: 1.438963E-10 1.438021E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 72442
Run: 100000
Dead: 28
ICR: 98230
OCR: 0
Accumulated BESSY: 59704.24033
Accumulated Ioni: 2.87680725E-08
N Accumulation: 200
Normfactor:       95968520.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Co
exp_x 72.0204010009766 72.0204
exp_y 122.265808105469 122.2658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  27   1      6.925   6140666.      2479.     133.38
