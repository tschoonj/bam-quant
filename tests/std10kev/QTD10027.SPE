$DATA:
0 2000
17087
11141
4952
1554
342
61
12
17
8
7
5
1
1
3
0
10
138
1414
1882
1663
1624
1731
1542
1480
1509
1423
1381
1317
1256
1245
1243
1224
1221
1169
1145
1138
1152
1051
1048
1062
963
1021
955
942
1017
928
915
983
920
880
918
873
838
826
889
941
846
879
899
873
874
857
810
803
845
767
810
744
792
851
773
766
830
781
793
770
768
758
780
718
739
722
751
745
722
745
692
703
737
740
808
750
816
736
738
732
728
731
772
760
816
771
779
786
748
837
751
786
667
747
742
768
706
705
715
680
669
696
686
699
665
718
704
679
687
679
694
708
738
741
754
794
718
743
637
703
646
679
688
697
666
737
775
863
942
1122
1164
1137
1041
917
790
732
701
716
680
694
647
694
714
723
634
694
649
662
620
650
602
609
633
668
628
688
628
604
636
633
645
608
626
612
685
694
645
670
699
678
695
633
640
655
665
669
642
650
660
662
653
657
643
652
643
652
670
656
676
676
661
708
671
657
668
715
639
685
691
702
671
677
699
659
713
725
699
713
707
737
700
684
757
690
710
725
746
769
764
708
740
802
745
784
829
857
855
824
853
802
835
838
911
1016
1140
1376
1777
2159
2590
3057
3201
3095
2802
2374
1961
1510
1217
1116
1050
1008
1044
1036
1070
1010
1026
1037
1008
1016
1023
1041
1064
1104
1009
1087
1139
1163
1149
1174
1256
1202
1211
1324
1348
1442
1543
1536
1563
1651
1559
1559
1470
1412
1434
1393
1378
1346
1409
1350
1390
1360
1406
1424
1525
1474
1539
1680
1680
1707
1881
1885
1991
2043
2158
2333
2289
2417
2460
2739
3028
3054
3329
3646
4139
4571
5142
6381
8427
12934
23374
44141
83180
148826
248977
377594
522224
655868
744362
765530
713838
601782
458385
316562
199443
114169
59516
28955
13614
6545
3626
2533
2062
1814
1721
1719
1805
1842
1874
2036
2331
2545
2917
3432
4097
5482
7558
12224
18896
29174
44180
62391
79983
94977
104033
103415
94493
78617
59584
41948
26627
15354
8368
4323
2034
938
499
246
171
137
109
115
109
109
105
97
86
80
73
78
51
70
65
60
67
69
63
54
49
55
47
47
38
33
28
41
36
43
34
49
34
44
37
48
48
52
46
40
33
40
30
26
22
36
24
37
40
19
23
37
26
21
23
19
20
24
21
20
22
31
23
20
15
11
16
18
25
18
14
28
23
21
19
28
20
21
29
33
29
41
27
38
43
41
32
49
31
56
65
68
85
116
135
165
206
221
262
292
352
386
375
359
337
310
270
201
195
129
104
53
57
46
23
20
13
21
23
12
9
13
11
12
16
7
18
25
10
15
17
12
12
15
10
13
7
13
9
11
9
18
14
10
15
8
17
3
18
13
9
19
6
10
13
16
21
17
13
13
12
9
8
11
8
8
16
8
10
7
14
10
9
13
16
8
19
13
23
15
13
10
17
12
10
11
17
14
11
19
15
18
13
13
12
14
21
21
13
22
17
26
28
27
39
28
39
45
45
30
33
32
22
22
23
27
16
12
22
18
16
9
13
17
19
15
15
16
17
17
20
18
17
20
19
24
24
27
20
36
27
24
26
29
21
25
26
21
17
27
19
25
21
9
28
17
20
19
19
20
19
26
27
31
32
21
31
28
25
30
45
33
33
43
57
64
61
65
80
80
134
161
265
377
628
857
1305
1793
2326
2934
3498
4015
4580
4545
4353
4110
3627
2904
2393
1713
1297
837
588
368
203
115
88
56
43
43
30
51
28
39
47
64
70
99
133
171
230
364
457
555
781
934
1007
1142
1250
1251
1171
1045
933
742
581
445
330
225
137
81
58
32
16
13
10
5
3
6
9
7
7
9
3
19
11
19
19
34
40
53
68
64
86
87
91
69
82
71
49
59
45
30
19
11
12
8
8
3
0
3
2
3
1
0
2
2
0
0
1
0
0
2
0
0
1
0
0
0
1
1
0
0
0
1
0
0
0
0
0
1
0
0
1
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
2
0
0
1
2
3
1
6
2
3
3
10
4
5
7
2
3
5
5
7
1
3
2
3
2
1
1
1
0
0
0
0
0
1
0
0
0
1
0
1
0
0
0
0
0
0
0
1
1
1
1
0
1
0
0
0
3
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
2
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
2
0
0
0
0
0
0
1
0
1
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
0
1
1
1
2
0
1
1
4
2
3
5
11
11
12
15
17
16
21
18
15
20
12
19
8
8
9
12
5
5
2
3
0
3
0
0
0
0
1
0
0
1
1
1
2
1
1
5
1
4
8
5
6
5
5
10
9
3
4
6
4
4
3
3
0
3
2
1
1
0
1
0
1
0
0
0
0
0
1
0
1
1
0
2
1
1
0
1
1
0
1
2
1
0
1
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std10kev\std10027.spe
Start: 21.07.2014 16:02:56
Stop: 21.07.2014 16:04:37
BESSY: 298.0013134 299.0730879
IONI: 1.438963E-10 1.438021E-10
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 72442
Run: 100000
Dead: 28
ICR: 98230
OCR: 0
Accumulated BESSY: 59704.24033
Accumulated Ioni: 2.87680725E-08
N Accumulation: 200
Normfactor:       95968520.
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Co
 
exp_x 72.0204010009766 72.0204
exp_y 122.265808105469 122.2658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 871.55 871.549999496
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 10 10.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 0.03
$PEAKS:   
      1
  27   1      6.925   6140666.      2479.     133.38
