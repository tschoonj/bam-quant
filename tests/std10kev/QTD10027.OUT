     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:27:02  บ   
     บ  Spectrum: QTD10027.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 - 500        5 iterations done    บ   
     บ  ChiSquare =   326.6   last change =   .00%    lambda= 1.E-07   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -72.8 ๑   100.0   บ     -72.8 ๑      .8   บ   
     บ  GAIN (eV/ch)  บ     20.357 ๑   2.000   บ    20.357 ๑    .002   บ   
     บ det NOISE (eV) บ     -114.7 ๑    40.0   บ    -114.7 ๑    -2.8   บ   
     บ  FANO factor   บ       .252 ๑    .050   บ      .252 ๑    .003   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Co-Ka                           6140666. ๑   2479.           บ   
     บ    KA1         6.925   1.00000     6140666. ๑   2480.    133.4  บ   
     บ              343.758    155.20        4265.           9.34E-01  บ   
     บ                                                                 บ   
     บ  2 Co-Kb                            892749. ๑    947.           บ   
     บ    KB1         7.649   1.00000      892749. ๑    948.     97.4  บ   
     บ              379.324    167.28        2235.           9.50E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                               0. ๑      0.           บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
